app/controller/controllers.py

```diff
@@ -9,4 +9,9 @@
 @get_index_bp.route('/')
 def index():
            # Render index.html
-           return render_template('index.html')
\ No newline at end of file
+           return render_template('index.html')
+
+@get_Sign_Up.route('/Sign_Up')
```

Comment:
Are you sure it's working ? I can't see `get_Sign_up` declared. You can take reference of `get_index_bp` above in line 7

Reply:
No, this is not quite correct, please declare the variables above, after line 7.